<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #000028;
color: white;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>474</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QFrame" name="horizontalFrame_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="ressource1.qrc">:/new/prefix1/Asset 1.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>531</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>650</width>
             <height>400</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <property name="spacing">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_video">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>400</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>650</width>
                <height>400</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="horizontalWidget_2" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>111</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="start_cam_button">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
              </property>
              <property name="text">
               <string>Start Camera</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="take_pic_button">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
              </property>
              <property name="text">
               <string>Take Picture</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_fire">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>45</height>
               </size>
              </property>
              <property name="text">
               <string>FIRE DETECTED</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_pump">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>45</height>
               </size>
              </property>
              <property name="text">
               <string>PUMP STARTED</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_notify">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>45</height>
               </size>
              </property>
              <property name="text">
               <string>ADMIN NOTIFIED</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_alarm">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>45</height>
               </size>
              </property>
              <property name="text">
               <string>ALARM FIRED</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>288</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>475</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QWidget" name="horizontalWidget_7" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>164</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="view_all_button">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
              </property>
              <property name="text">
               <string>View All</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-3</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-4</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-6</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-7</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: #000028;
font-weight: 800;</string>
             </property>
             <property name="text">
              <string>Bot Bk-8</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionResource">
   <property name="text">
    <string>Resource</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressource1.qrc"/>
 </resources>
 <connections/>
</ui>
